Android/ThirdPartyAuth/app/src/main/java/com/bentley/sample/thirdpartyauth/LoginViewModel.kt

```diff
@@ -0,0 +1,55 @@
+package com.bentley.sample.thirdpartyauth
+
+import android.content.Context
+import android.content.Intent
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.auth0.android.Auth0
+import com.auth0.android.authentication.AuthenticationException
+import com.auth0.android.callback.Callback
+import com.auth0.android.provider.WebAuthProvider
+import com.auth0.android.result.Credentials
+import com.bentley.sample.shared.MainActivity
+
+class LoginViewModel(
+	resourceHelper: IResourceHelper,
+	onAuth0TokenUpdated: (auth0Token: String)->Unit = {},
+) : ViewModel() {
+	/** Used to lookup string resources */
+	private val _resourceHelper = resourceHelper
+	var onAuth0TokenUpdated = onAuth0TokenUpdated
+
+	/** Authorization client to login to Auth0 */
+	private var _auth0Client: Auth0 = Auth0(
+		resourceHelper.getString(R.string.ITMSAMPLE_AUTH0_CLIENT_ID),
```

Comment:
I thought these settings moved into ITMAppConfig.json. The README.MD instructions certainly imply this, so I don't think this is the correct way to get the client ID and the domain.

Reply:
I setup the build to load the values in the ITMAppConfig.json to resource strings that are accessed in this way. itmsample.auth0,clie It seemed liked the cleanest way to do it. I will make this more clear in the readme and code comments